### DISCUSSION TITLE
What's the latest stable version for java 8?

### DISCUSSION THREAD

**Discussion by deerball** [optimatika/ojAlgo#522] [General, ojAlgo and related code]
My project is in java 8 code level and also I need to use ojAlgo in this project.

What's the latest stable version for java 8?

From the disscussion https://github.com/optimatika/ojAlgo/discussions/512, I got a message that version 51.4.1 is the last one supporting java 8. Is there an official announcement for it?

What's the difference between the latest version 53.0.0 and the latest stable version for java 8? Will the java 8 version keep updated?

**Comment by apete** [comment]
Yes, v51.4.1 is the latest that support Java 8. There are no plans for any releases supporting legacy versions. Currently targets Java 11. Will move up when there's a good reason.

Changes are logged in the [changelog](https://github.com/optimatika/ojAlgo/blob/develop/CHANGELOG.md).
